test/src/unit-current-domain-rest.cc

```diff
@@ -289,3 +375,122 @@ TEST_CASE_METHOD(
   REQUIRE(tiledb_array_close(ctx_c_, array) == TILEDB_OK);
   tiledb_array_free(&array);
 }
+
+TEST_CASE_METHOD(
+    RestCurrentDomainFx,
+    "C API: Current Domain basic schema evolution at timestamp",
+    "[current_domain][evolution][rest]") {
+  // Create array schema at ts=1
```

Comment:
Thank you for adding the test! REST CI is failing only for query v2, which is the old way we've been serializing queries and is not the default since a long time. There is a known issue on query v2 and time traveling (https://app.shortcut.com/tiledb-inc/story/49827/multiple-read-errors-on-a-particular-dataframe#activity-50249), so I wouldn't worry too much. I suggest to add a skip for the query v2 case so that the failure is avoided.